_posts/technotes/2017-09-07-spelling-release.md

```diff
@@ -0,0 +1,88 @@
+---
+name: spelling-release
+layout: post_discourse
+title: "Spelling 1.0: quick and effective spell checking in R"
+date: 2017-09-07
+authors:
+  - name: Jeroen Ooms
+categories: blog
+tags:
+- r
+- packages
+- software
+- spelling
+- text
+---
+
+
+The new rOpenSci [spelling](https://cran.r-project.org/web/packages/spelling/index.html) package provides utilities for spell checking common document formats including latex, markdown, manual pages, and description files. It also includes tools especially for package authors to automate spell checking of R documentation and vignettes.
+
+## Spell Checking Packages
+
+The main purpose of this package is to quickly find spelling errors in R packages. Use `spell_check_package()` function extracts all text from your package manual pages and vignettes, compares it against a language (e.g. en_US or en_GB), and lists potential errors in a nice tidy format:
+
+
+```r
+> spelling::spell_check_package("~/workspace/writexl")
+  WORD       FOUND IN
+booleans   write_xlsx.Rd:21
+xlsx       write_xlsx.Rd:6,18
+           title:1
+           description:1
+```
+
+Results may contain be false positives, i.e. names or technical jargon which does not appear in the English dictionary. Therefore the spelling package maintains a WORDLIST file, which serves as a package-specific dictionary of allowed words: 
+
+```r
+> spelling::update_wordlist("~/workspace/writexl")
```

Comment:
It may not be clear to the reader that `writexl` is a package and you're using it as an example. maybe say in the text, e.g., "Here, using `writexl` as an example"

Reply:
OK will do